update readme (#53)
worldline/RunningOrder: README.md

# Running Order

> A SwiftUI app for helping demonstrate sprint changes, for macOS.

Reviewer: ```suggestion
A macOS SwiftUI app to help demonstrate sprint changes.
```

![Image](Images/image1.png?)

## Goal

> When you work in a agile team, you need to show the changes of the finished sprint. This app is designed to help a development team to organize which feature to show, in which order, with some needed informations such as links, usernames, environment names,...

Reviewer: ```suggestion
When you work in an Agile team, you need to show the changes of the finished sprint. This app is designed to help a development team organize which feature to show, in which order, with the right information such as links, usernames, environment names, ...
```

> In order to work with multiple members of the team and respect users privacy, all the informations are stored in an iCloud private database owned by a member of the team.

Reviewer: ```suggestion
In order to collaborate with multiple members of the team and respect users privacy, all the information is stored in an iCloud private database owned by a member of the team.
```

## Roadmap
Currently this app is at a very early state of development. The first goal is to meet the requirements of one dev team at Worldline, be useful, and prove its value.

> The second goal is to be generalized to other iOS development teams, as the app requires a mac to work. Past this point, we'll see if we can launch the app on the mac app store, to reach development teams outside Worldline.

Reviewer: ```suggestion
The second goal is to be generalized to other iOS development teams, as the app requires a Mac to work. Past this point, we'll see if we can launch the app on the Mac App Store, to reach development teams outside Worldline.
```

Then, develop a counterpart to reach other interested teams (iOS version, Windows version, or web, in order to work on desired platforms)

## Technical requirements

> Currently the app supports macOS Catalina, but it will be upgraded to Big Sur soon to permit usage of newest SwiftUI changes.

Reviewer: ```suggestion
Currently the app only supports macOS Catalina, but it will be upgraded to Big Sur soon to allow the usage of the newest SwiftUI changes.
```

Author: seen together

> You'll need a iCloud account on the mac. (A cloudless functional build is not planned for now, but, it could be studied later)

Reviewer: ```suggestion
You'll need an iCloud account on the Mac. (A cloudless functional build is not planned for now, but could be studied later)
```
